feat: add `stats/incr/nankurtosis`

Girish-Garg opened the pull request:
Resolves #5561.

## Description

> What is the purpose of this pull request?

This pull request:

-   added incremental corrected sample excess kurtosis accumulator ignoring NaN by making a wrapper function over `@stdlib/stats/incr/kurtosis` main function.

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves #5561 

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

Test:
![image](https://github.com/user-attachments/assets/b1e4fb8f-db46-4082-94e0-f29e0242e45c)

Examples:
![image](https://github.com/user-attachments/assets/65cf808e-8716-4889-b3f0-39b4d83f91e7)

Benchmark:
![image](https://github.com/user-attachments/assets/4504ff24-8196-43f2-b6d9-622f572d9f3a)

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

Girish-Garg committed:
feat: add incremental corrected sample excess kurtosis accumulator ignoring NaN

Girish-Garg committed:
fix: changed the wrong svg

Girish-Garg committed:
Merge branch 'stdlib-js:develop' into feature/nankurtosis

23f3001304 commented:
/stdlib help

Girish-Garg committed:
chore: fix-linting issues and clean-up

Girish-Garg committed:
chore:fix

Girish-Garg committed:
chore: fix 2

Girish-Garg commented:
/stdlib update-copyright-years

stdlib-bot committed:
chore: update copyright years

kgryte commented:
@Girish-Garg You'll want to resolve the CI lint failure.

Planeshifter commented on lib/node_modules/@stdlib/stats/incr/nankurtosis/docs/types/index.d.ts:
```diff
@@ -0,0 +1,62 @@
+type accumulator = ( x?: number ) => number | null;
+
+/**
+* Returns an accumulator function which incrementally computes a corrected sample excess kurtosis.
```
Missing "ignoring `NaN` values".

```suggestion
* Returns an accumulator function which incrementally computes a corrected sample excess kurtosis, ignoring `NaN` values.
```

Planeshifter commented on lib/node_modules/@stdlib/stats/incr/nankurtosis/examples/index.js:
```diff
@@ -0,0 +1,43 @@
+	if ( i < 3 ) {
+		console.log( '%d\t%s', v.toFixed( 4 ), kurtosis );
+	} else {
+		console.log( '%d\t%d', v.toFixed( 4 ), ( kurtosis === null ) ? kurtosis : kurtosis.toFixed( 4 ) );
```
`.toFixed()` returns a string. Use `%s`, not `%d`. Same on line 38.

Planeshifter committed:
chore: address review feedback and update copyright year

- fix incorrect function name in `@example` and example script
- correct type checking note wording
- remove manually-populated related packages section
- use `format` for benchmark name string concatenation
- update copyright year to 2026

Planeshifter commented:
/stdlib merge

stdlib-bot committed:
Merge remote-tracking branch 'upstream/develop' into feature/nankurtosis

Planeshifter committed:
chore: address review feedback

- require `@stdlib/stats/incr/kurtosis` package root instead of deep `/lib` path
- add blank line before repl.txt See Also heading
- append ", ignoring `NaN` values" to factory description in index.d.ts
